Installer/Program.cs

```diff
@@ -266,12 +270,32 @@ private static int BuildMsiPackage(MsiOptions opts)
             new RegValue(RegistryHive, RegistryKey, "Manager:LogFileLocation",
                 @"[INSTALLFOLDER]coder-desktop-service.log"),
             new RegValue(RegistryHive, RegistryKey, "Manager:TunnelBinarySignatureSigner", "Coder Technologies Inc."),
-            new RegValue(RegistryHive, RegistryKey, "Manager:TunnelBinaryAllowVersionMismatch", "false"),
-            // Add registry values that are consumed by the App MutagenController. See App/Services/MutagenController.cs
-            new RegValue(RegistryHive, RegistryKey, "AppMutagenController:MutagenExecutablePath",
-                @"[INSTALLFOLDER]mutagen.exe")
+            new RegValue(RegistryHive, RegistryKey, "Manager:TunnelBinaryAllowVersionMismatch", "false")
         );
 
+        // rather than adding individual reg values, it's much clearer to write the whole config as JSON then
+        // programmatically spit out the registry values.
+        var appConfigJson = """
+                            {
+                                "MutagenController": {
+                                    "MutagenExecutablePath": "[INSTALLFOLDER]mutagen.exe"
+                                },
+                                "Serilog": {
+                                    "Using": ["Serilog.Sinks.File"],
+                                    "MinimumLevel": "Information",
+                                    "Enrich": [ "FromLogContext" ],
+                                }
+                            }
+                            """;
```

Comment:
if we're cool with this style of configuration, I can also apply it to the VPN service.